Source/Scene/Model.js

```diff
@@ -1010,7 +1014,7 @@ define([
             setCachedGltf(model, cachedGltf);
             gltfCache[cacheKey] = cachedGltf;
 
-            loadArrayBuffer(url, options.headers).then(function(arrayBuffer) {
+            RequestScheduler.request(url, loadArrayBuffer, options.headers).then(function(arrayBuffer) {
```

Comment:
I think I mentioned this in a past review, but `Model` may be making requests on behalf of 3D Tiles so I think we should allow an override for the request type via a `@private` options parameter on the constructor.